com.haysell.pos.yaml

```diff
@@ -0,0 +1,23 @@
+id: com.haysell.pos
+runtime: org.kde.Platform
+runtime-version: '5.15-23.08'
+sdk: org.kde.Sdk
+command: haysell-pos
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+  - --filesystem=host
+cleanup:
+  - '/include'
+  - '*.a'
+modules:
+  - name: haysell-pos
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=Release
+    sources:
+      - type: git
+      url: https://github.com/Delphi251189/haysell-pos.git
```

Comment:
this manifest is invalid (indent matter in yaml and I'm sure `flatpak-builder` generated at least a warning). Alst git sources REQUIRE a commit or tag.